chanzuckerberg/single-cell-data-portal
# fix: 1426 set new explorer_url for cloned datasets post revision creation

### Reviewers
**Functional:** 
@MDunitz 

**Readability:** 
@leslieypark 

Ticket [#1426](https://app.zenhub.com/workspaces/single-cell-5e2a191dad828d52cc78b028/issues/chanzuckerberg/single-cell-data-portal/1426)

---

## Changes
- Sets `explorer_url=None` when cloning a dataset after a revision. This is important because otherwise looking up for `explorer_url` (which is done by the meta endpoint) will yield multiple results and might not return the correct one.

## QA steps (optional)
QA this in staging by: 
1. Create a collection, add a dataset, publish
2. Create a revision
3. Open the old dataset in the explorer
4. Verify that the `dataset_identifier` in the `config` payload is the same as its id

## TODO
Add a condition to the revision functional test
